### PR TITLE
Change the topic name of the control_cmd to align them in trajectory_follower_node

### PR DESCRIPTION
**Pull request by rej55**
## Description

Rename topic names in trajectory_follower_nodes.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [pull request guidelines][pull-request-guidelines]
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: control/trajectory_follower_nodes/src/lateral_controller_node.cpp [hunk 1]
```diff
@@ -146,20 +146,20 @@ LateralController::LateralController(const rclcpp::NodeOptions & node_options)
 
   m_pub_ctrl_cmd =
     create_publisher<autoware_auto_control_msgs::msg::AckermannLateralCommand>(
-    "output/lateral/control_cmd", 1);
+    "~/output/lateral/control_cmd", 1);
```
Comment (maxime-clem) [inline control/trajectory_follower_nodes/src/lateral_controller_node.cpp:149]: If we use the private namespace (with `~`), the topic will look like `/namespace/lateral_controller_node/output/lateral/control_cmd`. I think in that case it is not necessary to specify `lateral`.
```suggestion
    "~/output/control_cmd", 1);
```
Same suggestion for the other topics and for the longitudinal controller node.

**Review by maxime-clem — COMMENTED** [review]
Looks good. My only comment is about the use of private namespace (`~`).

File: control/trajectory_follower_nodes/src/lateral_controller_node.cpp (hunk 1 above)
Comment (rej55) [reply control/trajectory_follower_nodes/src/lateral_controller_node.cpp:149]: I think your suggestion looks nice.
I'll change them.

**Review by maxime-clem — APPROVED** [review]

**Comment by TakaHoribe** [comment]
@rej55 We are going to apply the same changes to the .auto, so please care about the PR title since it will be a commit message. This time, something like "Change the topic name of the control_cmd to align them in trajectory_follower_node" is easier to understand.

**Pull request merged**